src/components/application_manager/test/resumption/resumption_data_test.cc

```diff
@@ -536,6 +539,7 @@ void ResumptionDataTest::SetChoiceSet() {
   for (uint32_t i = 0; i < kCountOfChoiceSets_; ++i) {
     for (uint32_t j = 0; j < kCountOfChoice_; ++j) {
       char numb[12];
+      // cppcheck-suppress invalidPrintfArgType_sint
```

Comment:
@Ypostolov Do not suppress this issue. You can find solution at several comment above.